content/kubeflow/install.md

```diff
@@ -5,31 +5,37 @@ weight: 10
 draft: false
 ---
 
-In this chapter, we will install Kubeflow on Amazon EKS cluster. The cluster creation steps are outlined in [Launch EKS](/eksctl/launcheks/) chapter.
+In this chapter, we will install Kubeflow on Amazon EKS cluster. If you don't have an EKS cluster, please follow instructions from [getting started guide] (/prerequisites) and then launch your EKS cluster using [eksctl](/eksctl) chapter
 
 ### Install Kubeflow on Amazon EKS
 
-Download 0.6.1+ release of `kfctl`. This binary will allow you to install Kubeflow on Amazon EKS:
+Download 0.7 RC6 release of `kfctl`. This binary will allow you to install Kubeflow on Amazon EKS:
```

Comment:
can we make this `RC6+`?

Reply:
Looks like Kubeflow will move out of RC over the weekend. I'll keep an eye and make a change once they update release